### ISSUE TITLE
[Accessibility] Keyboard cannot check/uncheck listView items when enabling listView checkBox property in Virtual Mode

### ISSUE THREAD

**Issue by Olina-Zhang** [dotnet/winforms#4033]
* .NET Core Version: 
.Net SDK 5.0.100-rc.2.20477.2

* Have you experienced this same bug with .NET Framework?: 
Yes

**Minimal repro:**
1. Open the attached ListViewInVirtualMode.zip application in VS, then build and run it.
[ListViewInVirtualMode.zip](https://github.com/dotnet/winforms/files/5303187/ListViewInVirtualMode.zip)
2. Focus on ListView item, press Space to check/uncheck it in its checkBox

**Actual behavior:**
Keyboard cannot check/uncheck listView items' checkBox by pressing "Space" in ListView virtual mode, and for mouse, need double-click to check/uncheck, not single-click.
![ListViewVirtualModeIssue](https://user-images.githubusercontent.com/26474449/94645595-cf7e5f80-031e-11eb-8a94-eed6ba558aae.gif)

**Expected behavior:**
Keyboard can check/uncheck listView items' checkBox by pressing "Space" in ListView virtual mode, like the result in default mode
![ListViewDefaultMode](https://user-images.githubusercontent.com/26474449/94645601-d2795000-031e-11eb-8679-ac2f9316cd4d.gif)

**Comment by SergeySmirnov-Akvelon** [comment]
Based on the investigation above and the fact that the `Click` and `KeyPress` methods initially contained checks for virtual mode, it was decided to consider the current behavior as expected.
